expression/builtin_miscellaneous_vec.go

```diff
@@ -411,11 +411,73 @@ func (b *builtinTimeAnyValueSig) vecEvalTime(input *chunk.Chunk, result *chunk.C
 }
 
 func (b *builtinInetAtonSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinInetAtonSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+	var (
+		byteResult, res uint64
+		dotCount        int
+	)
+	result.ResizeInt64(n, false)
+	i64s := result.Int64s()
+	result.MergeNulls(buf)
+	for i := 0; i < n; i++ {
+		ipAddr := buf.GetString(i)
```

Comment:
I think we should check `result.IsNull(i)` first.

Reply:
That make sense. I will change it accordingly.